cmd/helmExecute.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/SAP/jenkins-library/pkg/kubernetes"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+)
+
+func helmExecute(config helmExecuteOptions, telemetryData *telemetry.CustomData) {
+	utils := kubernetes.NewDeployUtilsBundle()
+
+	// error situations should stop execution through log.Entry().Fatal() call which leads to an os.Exit(1) in the end
+	if config.DeployTool == "helm" || config.DeployTool == "helm3" {
```

Comment:
I think we should only support `helm3` and not support old helm versions with this step

Reply:
done